### PR TITLE
getting-started: Add SRC and Audio compress for i.MX user guide

### PR DESCRIPTION
**Pull request by dbaluta**
This describes how to use SRC and Audio compress features with i.MX8.

Signed-off-by: Daniel Baluta <daniel.baluta@nxp.com>

### CODE REVIEW

**Comment by dbaluta** [comment]
@lgirdwood i noticed that the https://thesofproject.github.io/latest/getting_started/index.html is not updated. Do I need to ping someone for this?

File: getting_started/nxp/sof_imx_user_guide.rst [hunk 1]
```diff
@@ -244,3 +244,102 @@ After, booting we will see now that SOF sound card will have two subdevices:
...
+   CONFIG_CADENCE_CODEC_AAC_DEC=y
+   CONFIG_CADENCE_CODEC_AAC_DEC_LIB="/path/to/aac/library"
+   
+   # Enable MP3 Cadence decoder
+   CONFIG_CADENCE_CODEC_MP3_DEC=y
+   CONFIG_CADENCE_CODEC_MP3_DEC_LIB="/path/to/mp3/library"
+
+Contact NXP Tech support for information on how to obtain Cadence proprietary algorithms.
+
+As, usual we will boot the i.MX8QM board using imx8qm-mek-sof-wm8960.dtb. Let's see an example on how to test
```
Comment (iuliana-prodan) [inline getting_started/nxp/sof_imx_user_guide.rst:330]: same as above: s/As, usual/As usual,

**Review by iuliana-prodan — CHANGES_REQUESTED** [review]

File: getting_started/nxp/sof_imx_user_guide.rst (hunk 1 above)
Comment (dbaluta) [reply getting_started/nxp/sof_imx_user_guide.rst]: fixed

File: getting_started/nxp/sof_imx_user_guide.rst [hunk 2]
```diff
@@ -244,3 +244,102 @@ After, booting we will see now that SOF sound card will have two subdevices:
...
+|  toolchain    |      direction     |          input rate (kHz)                          | output rate  (kHz) |
++===============+====================+====================================================+====================+
+|     GCC       | playback/capture   |  8 16 32 44.1 48 96                                |         48         |
++---------------+--------------------+----------------------------------------------------+--------------------+
+|     XCC       |      playback      |  8 11.025 16 22.05 32 44.1 48 64 88.2 96 176.4 192 |         48         |
++---------------+--------------------+----------------------------------------------------+--------------------+
+|     XCC       |      capture       |  8 11.025 16 22.050 32 44.1 48                     |         48         |
++---------------+--------------------+----------------------------------------------------+--------------------+
+
+As, usual we will boot the i.MX8QM board using imx8qm-mek-sof-wm8960.dtb. We need to use sof-imx8-src-wm8960.tplg topology file.
```
[the remark this answers is not in the document]
Comment (dbaluta) [reply getting_started/nxp/sof_imx_user_guide.rst]: fixed

**Review by iuliana-prodan — APPROVED** [review]

**Comment by lgirdwood** [comment]
> @lgirdwood i noticed that the https://thesofproject.github.io/latest/getting_started/index.html is not updated. Do I need to ping someone for this?

I think @deb-intel will publish when she's back from vacation, she usually does a grammar check/fixup prior to publishing.

**Review by lgirdwood — APPROVED** [review]

File: getting_started/nxp/sof_imx_user_guide.rst [hunk 3]
```diff
@@ -244,3 +244,102 @@ After, booting we will see now that SOF sound card will have two subdevices:
...
+Supported codecs on i.MX8QM:
+
++---------------+--------------------+----------------+-------------------------------------------------+
+|   codec       |              topology               |           Test command                          |
++===============+=====================================+=================================================+
+|    PCM        | sof-imx8-processing-pcm-wm8960.m4   | cplay -c 1 -d 0 -f 2 -b 7680 -I PCM sample.wav  |
++---------------+-------------------------------------+-------------------------------------------------+
+|    MP3        | sof-imx8-processing-mp3-wm8960.m4   | cplay -c 1 -d 0 -f 2 -b 7680 -I MP3 sample.mp3  |
++---------------+-------------------------------------+-------------------------------------------------+
+|    AAC        | sof-imx8-processing-aac-wm8960.m4   | cplay -c 1 -d 0 -f 2 -b 7680 -I MP3 sample.aac  |
```
Comment (paulstelian97) [inline getting_started/nxp/sof_imx_user_guide.rst:307]: "-I MP3" Is this intended?

Comment (dbaluta) [reply getting_started/nxp/sof_imx_user_guide.rst:307]: @paulstelian97 thanks it's a typo. Will fix it with next batch of changes for documentation.

**Comment by dbaluta** [comment]
@deb-intel can you please publish the latest sources?

**Comment by deb-intel** [comment]
@dbaluta Yes! I will begin the review process now and hopefully publish by the end of my day or early tomorrow (U.S, Eastern).

**Comment by deb-intel** [comment]
@dbaluta The SOF website is published with the .MX additions.

**Pull request merged**